go/billing/tasks.py

```diff
@@ -375,7 +375,8 @@ def generate_monthly_statement(account_id, from_date, to_date):
         title=settings.MONTHLY_STATEMENT_TITLE,
         type=Statement.TYPE_MONTHLY,
         from_date=from_date,
-        to_date=to_date)
+        to_date=to_date,
+        status=Statement.STATUS_STATEMENT_PENDING)
```

Comment:
Monkey-patch `get_transactions()` or `generate_statement_items()` with something that fails and assert that you end up with a pending statement?